obspy/core/trace.py

```diff
@@ -2496,8 +2496,11 @@ def times(self, type="relative", reftime=None):
                 [self.stats.starttime + t_ for t_ in time_array])
         elif type == "matplotlib":
             from matplotlib.dates import date2num
-            time_array = date2num([(self.stats.starttime + t_).datetime
-                                   for t_ in time_array])
+            SECONDSPERDAY = 86400
```

Comment:
Having this as an extra variable is probably not necessary as most people should recognize that numbers and its only used once in the next name.

Reply:
Thanks for your suggestions. Requested changes have been made.